src/utils/comparison-load-manager.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { ComparisonLoadManager } from "./comparison-load-manager";
+
+let comparisonLoadManager;
```

Comment:
* Set type of this variable and check only public members.
* Test particular use cases we have in the application: start loading, resolve sides one by one, check public results.
* Improve coverage if necessary
* Check Event